duktape/src/main/jni/DuktapeContext.cpp

```diff
@@ -93,6 +93,45 @@ duk_ret_t javaObjectFinalizer(duk_context *ctx) {
   return 0;
 }
 
+void handleJavaScriptError(JNIEnv* env, duk_context* ctx) {
```

Comment:
Can the name of this method be changed to suggest its side effect? Like `queueJavaExceptionForJavaScriptError` ? Hard to figure out a word that means `env->Throw` where the exception _will_ throw, but not til JNI returns.